<launch>
    <rosparam file="$(find innopolis_vtol_dynamics)/config/uavcan_communicator.yaml" command="load" ns="uavcan/" />
    <node pkg="uavcan_communicator" type="uavcan_communicator" name="px4_uavcan_communicator" output="screen"/>
</launch>
